### PR TITLE
Add CSS constants file

### PR DESCRIPTION
**Pull request by jreis13**
Why:
* We want to be able to access some variables globally.

How:
* Created and defined variables in a globals.css file.

### CODE REVIEW

File: frontend/src/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import ReactDOM from "react-dom";
 import { BrowserRouter } from "react-router-dom";
 
 import "./index.css";
+import "styles";
```
Comment (mirhamasala) [inline frontend/src/index.js:5]: This doesn't do much except breaking the build. Did your editor not show an error?

Comment (mirhamasala) [reply frontend/src/index.js:5]: You can also check the [logs](https://app.netlify.com/sites/peppy-peony-33d6b6/deploys/62e7f78973d2d10008fb83e2)

Comment (mirhamasala) [reply frontend/src/index.js:5]: Screenshot 2022-08-02 at 07 41 09

Comment (mirhamasala) [reply frontend/src/index.js:5]: What you want is to import these globals into the top level CSS file `index.css`


Which you can do so: `@import "styles/globals.css";`

File: frontend/src/styles/globals.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  --color-hot-pink: #ea1889;
+  --color-light-pink: #d396ed;
+  --color-green: #57f2b9;
+  --color-yellow: #fff0ad;
+  --color-black: #000;
+  --color-dark-gray: #404040;
+  --color-gray: #808080;
+  --color-light-gray: #bfbfbf;
+  --color-white: #fff;
+  --font-family-body: 'Work Sans';
```
Comment (mirhamasala) [inline frontend/src/styles/globals.css:13]: Have you configured your editor to apply Prettier on save? Seeing single quotes here.

Comment (mirhamasala) [reply frontend/src/styles/globals.css:13]: Also do this font families work like this? Do they have to be imported?

**Comment by mirhamasala** [comment]
Some general comments:

- If your build is failing, fix it, before asking for a review
- You can clean up your commit history before asking for a review. For example, the last three commits, you could've squashed into `Add CSS constants file` with `git rebase -i main`  - We can work on it together if you forgot how.

Screenshot 2022-08-02 at 08 00 19

File: frontend/src/index.js (hunk 1 above)
Comment (jreis13) [reply frontend/src/index.js:5]: Got it

File: frontend/src/styles/globals.css (hunk 2 above)
Comment (jreis13) [reply frontend/src/styles/globals.css:13]: Prettier should be applying on save but it's not working since yesterday I believe for some unknown reason

**Review by mirhamasala — APPROVED** [review]

**Pull request merged**